java-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/typesolvers/JavaParserTypeSolver.java

```diff
@@ -129,26 +142,24 @@ private CompilationUnit parse(File srcFile) throws FileNotFoundException {
             }
 
             File srcFile = new File(filePath);
-            try {
-                CompilationUnit compilationUnit = parse(srcFile);
-                Optional<com.github.javaparser.ast.body.TypeDeclaration<?>> astTypeDeclaration = Navigator.findType(compilationUnit, typeName);
-                if (astTypeDeclaration.isPresent()) {
-                    return SymbolReference.solved(JavaParserFacade.get(this).getTypeDeclaration(astTypeDeclaration.get()));
+            {
```

Comment:
I added this brace to limit the scope that local variables are visible in. I think this makes the code more readable because you know that you don't have to pay attention to anything declared inside that scope when you go on reading.